tokio/src/net/tcp/stream.rs

```diff
@@ -190,7 +190,7 @@ impl TcpStream {
     pub fn from_std(stream: std::net::TcpStream) -> io::Result<TcpStream> {
         let io = mio::net::TcpStream::from_std(stream);
         let io = PollEvented::new(io)?;
-        Ok(new_instrumented_resource!(Network, TcpStream { io }))
+        Ok(new_instrumented_resource!("network", TcpStream { io }))
```

Comment:
again, take it or leave it, but maybe
```suggestion
        Ok(new_instrumented_resource!("net", TcpStream { io }))
```